data-raw/i18n_translations.yml

```
@@ -408,6 +408,64 @@ text:
       또는 <code>&quot;</code>, <code>'</code>, <code>(</code>
       , <code>{</code>로 시작하는 구문을 닫는 <code>&quot;</code>, <code>'</code>,
       <code>)</code>, <code>}</code>을 잊었을 수도 있습니다.
+  unparsablequotes:
+    en: >
+      <p>It looks like your R code contains specially formatted quotation marks
+      or &quot;curly&quot; quotes (<code>{{character}}</code>)
+      around character strings, making your code invalid.
+      R requires character values to be contained in straight quotation
+      marks (<code>&quot;</code> or <code>'</code>).</p>
+      {{code}}
+      <p>Don't worry, this is a common source of errors when you copy code from
+      another app that applies its own formatting to text.
+      You can try fixing the code on that line with the following.
```

Sorry, I should have said "try replacing"

```suggestion
      You can try replacing the code on that line with the following.
```

---

be3813e